src/features/ScreenLock/ScreenLockForgotPassword.tsx

```diff
@@ -1,81 +0,0 @@
-import React, { FC, useContext } from 'react';
-
-import { Button } from '@mycrypto/ui';
-import { RouteComponentProps, withRouter } from 'react-router-dom';
-import styled from 'styled-components';
-
-import mainImage from '@assets/images/icn-forgot-password.svg';
```

Comment:
This asset is not used anymore

Reply:
Correct, if you search in PR you can see the `images/icn-forgot-password.svg` being removed.
Since GH folds the display it's not obvious.

Reply:
My mistake, sorry for the inconvenience.